packages/form-config/src/components/Antd/FileList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,282 @@
+    </Space>,
+    <Space
+      style={
+        Object {
+          "float": "right",
+          "marginBottom": 16,
+        }
+      }
+    >
+      <ForwardRef(SettingOutlined) />
+    </Space>,
+    <Table
+      columns={
+        Array [
+          Object {
+            "dataIndex": "identifier",
+            "ellipsis": true,
+            "key": "identifier",
+            "sortOrder": undefined,
+            "sorter": [Function],
+            "title": "Identifier",
+          },
+          Object {
+            "dataIndex": "label",
+            "ellipsis": true,
+            "key": "label",
+            "sortOrder": undefined,
+            "sorter": [Function],
+            "title": "File Name",
+          },
+          Object {
+            "dataIndex": "version",
+            "ellipsis": true,
+            "key": "version",
+            "sortOrder": undefined,
+            "sorter": [Function],
+            "title": "File Version",
+          },
+          Object {
+            "dataIndex": "createdAt",
+            "ellipsis": true,
+            "key": "createdAt",
+            "render": [Function],
+            "sortOrder": undefined,
+            "sorter": [Function],
+            "title": "Created At",
+          },
+          Object {
+            "dataIndex": "module",
+            "key": "module",
+            "title": "Module",
+          },
+          Object {
+            "key": "action",
+            "render": [Function],
+            "title": "Action",
+          },
+        ]
+      }
+      dataSource={
+        Array [
+          Object {
+            "createdAt": "Jun 19, 2020, 12:31:37 PM",
+            "form_relation": "",
+            "id": "52",
+            "identifier": "test-form-1.json",
+            "isDraft": false,
+            "isJsonValidator": false,
+            "jursdiction": "",
+            "label": "test form",
+            "module": "",
+            "version": "1.0.26",
+          },
+          Object {
+            "createdAt": "Jun 19, 2020, 4:23:22 PM",
+            "form_relation": "",
+            "id": "53",
+            "identifier": "reveal-test-file.json",
+            "isDraft": false,
+            "isJsonValidator": false,
+            "jursdiction": "",
+            "label": "test publish",
+            "module": "",
+            "version": "1.0.27",
+          },
+        ]
+      }
+      onChange={[Function]}
+      pagination={
+        Object {
+          "defaultPageSize": 5,
+          "pageSizeOptions": Array [
+            "5",
+            "10",
+            "20",
+            "50",
+            "100",
+          ],
+          "showQuickJumper": true,
+          "showSizeChanger": true,
+        }
+      }
+      rowKey="key"
+    />,
+  ],
+}
+`;
+
+exports[`components/Antd/FileList renders correctly for view file: title 1`] = `
+Object {
+  "children": "Releases: 1.0.11",
+  "level": 3,
+}
+`;
+
+exports[`components/Antd/FileList renders correctly: card 1`] = `
+Object {
```

Comment:
big snapshot

Reply:
@p-netm Tend to disagree its only 128 lines long and a snapshot of the props

Reply:
@p-netm Let me check if I can have it smaller